Tools-Override/tests.targets

```diff
@@ -125,25 +125,25 @@
 
   <!-- We need to generate a simple config file for desktop test executors to tell them to use DEVPATH environment variable to use their dependencies from there.
   DEVPATH is being set for desktop runs in RunTests.cmd to the TestHostPath. We need this approach to have desktop Helix runs. -->
-  <Target Name="GenerateDesktopExecutorsConfigFiles"
+  <Target Name="CopyDesktopExecutorsConfigFiles"
           BeforeTargets="GenerateTestExecutionScripts"
           Condition="'$(BuildingNETFxVertical)' == 'true'">
     <PropertyGroup>
+      <GenericConfigSourceFile>$(ToolsDir)\DesktopRunnerConfigFile.config</GenericConfigSourceFile>
```

Comment:
This should be override-able by individual test projects so please a d a condition that checks if it is already set or not. I would also rename it to NETFxTestRunnerAppConfig, or something along those lines.